test/tags/function-definitions.R

```diff
@@ -1,14 +1,38 @@
 fn <- function() {
-# <- definition.function  
+# <- definition.function
```

Comment:
oh, in-browser VS editor made these WS changes, are they important?

Reply:
I don't think so! I must have had whitespace trimming off by accident when I touched these last. Fine as is